partition_snapshot_row_cursor.hh

```diff
@@ -306,35 +434,44 @@ public:
     // Assumes that rows are not inserted into the snapshot (static). They can be removed.
     bool maybe_refresh_static() {
         if (!iterators_valid()) {
-            return maybe_advance_to(_position);
+            return maybe_advance_to(to_query_domain(_position));
```

Comment:
Would it make sense to replace `to_query_domain(_position)` with `position()`?

Reply:
Yes